data_management/services/tableau.py

```
@@ -31,6 +31,16 @@ def load_biopsy_tracking(self):
             records_modified = records_modified + 1
         return records_modified
 
+    def load_biopsy_tracking_long(self):
+        self.truncate_biopsy_tracking()
+        bt_results = self.dlu_management.get_biopsy_tracking_long()
+        query = "INSERT INTO biopsy_tracking_long(redcap_id, specimen_id, dlu_packageType, status) VALUES(%s, %s, %s, %s)"
+        records_modified = 0
+        for result in bt_results:
+            insert_result = self.db_tableau.insert_data(query, tuple(result.values()))
```

Ruff (0.14.14)

[error] Local variable `insert_result` is assigned to but never used

Remove assignment to unused variable `insert_result`

(F841)